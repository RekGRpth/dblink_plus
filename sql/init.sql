SET client_min_messages = warning;
\set ECHO none
\i dblink_plus.sql
\set ECHO all
RESET client_min_messages;
