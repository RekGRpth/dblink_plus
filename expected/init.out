SET client_min_messages = warning;
\set ECHO none
RESET client_min_messages;
\! sh regress_init.sh
